pages/common/openttd.md

```diff
@@ -0,0 +1,28 @@
+# openttd
+
+> Open source clone of the Microprose game “Transport Tycoon Deluxe".
+> More information: <https://www.openttd.org>.
+
+- Load new/save game at start:
+
+`openttd -g {{path/to/file}}`
+
+- Start with a set resolution (eg. 1920x1200):
+
+`openttd -r {{resolution}}`
+
+- Start with a different configuration file:
+
+`openttd -c {{path/to/file}}`
+
+- Start with selected drivers:
+
+`openttd -v {{video_driver}} -s {{sound_driver}} -m {{music_driver}}`
```

Comment:
Could you give an example of each driver here?

Reply:
Given that this command doesn't support long options, let's not do this, actually, because the placeholder words clarify which option controls which drivers.

Reply:
sure:
```bash
$ openttd -h
...
List of music drivers:
        fluidsynth: FluidSynth MIDI Driver
           extmidi: External MIDI Driver
              null: Null Music Driver

List of sound drivers:
               sdl: SDL Sound Driver
              null: Null Sound Driver

List of video drivers:
        sdl-opengl: SDL OpenGL Video Driver
               sdl: SDL Video Driver
         dedicated: Dedicated Video Driver
              null: Null Video Driver
```